website/docs/installation/air-gapped.mdx

```diff
@@ -15,7 +15,7 @@ To disable these outbound connections, set the following in your `.env` file:
 
 ## Configuration options
 
-To see a list of all configuration options, see [here](./configuration.mdx).
+To view a list of all configuration options, refer to [this page](./configuration.mdx).
```

Comment:
Thanks for this contribution @4d62 . To go full-bore tech-writing-editor, the best practice is to link to the title of the document, and never say "here" (as you corrected) nor even "this page".  (My editors at IBM and BMC were amazing.)

So something like "To view a list of all configuration options, refer to the [Configuration](./configuration.mdx) documentation."